velox/exec/HashProbe.cpp

```diff
@@ -540,21 +557,98 @@ void HashProbe::fillFilterInput(vector_size_t size) {
       filterInput_);
 }
 
+bool HashProbe::testFilterOnBuildSide(vector_size_t i, bool nullKeyRowsOnly) {
```

Comment:
If there are N rows in the hash table and we evaluate testFilterOnBuildSide on M probe rows, we'll copy N rows from hash table into vectors M times, right? Since copying is expensive, would it make sense to restructure the code so that we copy hash table only once?

Reply:
Is it safe to keep all rows from row container at same time?